snmp/datadog_checks/snmp/data/profiles/_juniper-cos.yaml

```diff
@@ -0,0 +1,183 @@
+    - OID: 1.3.6.1.4.1.2636.3.15.10.1.21
+      name: jnxCosIfsetQstatHpTcpRedDropPkts
+    # The total number of bytes RED-dropped at the output on the given interface.
+    - OID: 1.3.6.1.4.1.2636.3.15.10.1.23
+      name: jnxCosIfsetQstatTotalRedDropBytes
+    # The total number of low PLP non TCP bytes RED-dropped at the output on the given interface.
+    - OID: 1.3.6.1.4.1.2636.3.15.10.1.25
+      name: jnxCosIfsetQstatLpNonTcpRedDropBytes
+    # The total number of low PLP TCP bytes RED-dropped at the output on the given interface.
+    - OID: 1.3.6.1.4.1.2636.3.15.10.1.27
+      name: jnxCosIfsetQstatLpTcpRedDropBytes
+    # The total number of high PLP non TCP bytes RED-dropped at the output on the given interface.
+    - OID: 1.3.6.1.4.1.2636.3.15.10.1.29
+      name: jnxCosIfsetQstatHpNonTcpRedDropBytes
+    # The total number of high PLP TCP bytes RED-dropped at the output queue on the given interface.
+    - OID: 1.3.6.1.4.1.2636.3.15.10.1.31
+      name: jnxCosIfsetQstatHpTcpRedDropBytes
+    # The total number of low PLP packets RED-droppped at the output on the given interface.
+    - OID: 1.3.6.1.4.1.2636.3.15.10.1.33
+      name: jnxCosIfsetQstatLpRedDropPkts
+    # The total number of medium-low PLP packets RED-droppped at the output on the given interface.
+    - OID: 1.3.6.1.4.1.2636.3.15.10.1.35
+      name: jnxCosIfsetQstatMLpRedDropPkts
+    # The total number of medium-high PLP packets RED-droppped at the output on the given interface.
+    - OID: 1.3.6.1.4.1.2636.3.15.10.1.37
+      name: jnxCosIfsetQstatMHpRedDropPkts
+    # The total number of high PLP packets RED-droppped at the output on the given interface.  
+    - OID: 1.3.6.1.4.1.2636.3.15.10.1.39
+      name: jnxCosIfsetQstatHpRedDropPkts
+    # The total number of low PLP TCP bytes RED-dropped at the output queue on the given interface.
+    - OID: 1.3.6.1.4.1.2636.3.15.10.1.41
+      name: jnxCosIfsetQstatLpRedDropBytes
+    # The total number of medium-low PLP TCP bytes RED-dropped at the output queue on the given interface.
+    - OID: 1.3.6.1.4.1.2636.3.15.10.1.43
+      name: jnxCosIfsetQstatMLpRedDropBytes
+    # The total number of medium-high PLP TCP bytes RED-dropped at the output queue on the given interface.
+    - OID: 1.3.6.1.4.1.2636.3.15.10.1.45
+      name: jnxCosIfsetQstatMHpRedDropBytes
+    # The total number of high PLP TCP bytes RED-dropped at the output queue on the given interface.
+    - OID: 1.3.6.1.4.1.2636.3.15.10.1.47
+      name: jnxCosIfsetQstatHpRedDropBytes
+    # The total number of rate-limit dropped packets at the output on the given interface.
+    - OID: 1.3.6.1.4.1.2636.3.15.10.1.49
+      name: jnxCosIfsetQstatRateLimitDropPkts
+    # The total number of rate-limit dropped bytes at the output on the given interface.
+    - OID: 1.3.6.1.4.1.2636.3.15.10.1.51
+      name: jnxCosIfsetQstatRateLimitDropBytes
+    metric_tags:
+    - column:
+        OID:  1.3.6.1.4.1.2636.3.15.10.1.1
```

Comment:
is it possible to use the interface name from IF-MIB ?
```yaml
 - column:
      OID: 1.3.6.1.2.1.31.1.1.1.1
      name: ifName
    table: ifXTable
    tag: interface
  - column:
      OID: 1.3.6.1.2.1.31.1.1.1.18
      name: ifAlias
    table: ifXTable
    tag: interface_alias
```

Reply:
Done